server/src/main/java/com/cloud/api/query/dao/NetworkOfferingJoinDao.java

```diff
@@ -27,9 +27,9 @@
 
 public interface NetworkOfferingJoinDao extends GenericDao<NetworkOfferingJoinVO, Long> {
 
-    List<NetworkOfferingJoinVO> findByDomainId(long domainId);
+    List<NetworkOfferingJoinVO> findByDomainId(long domainId, Boolean includeAllDomainOffering);
```

Comment:
can you please add javadoc to explain the parameters here. I found them not to be intuitive with the implementation in the dao impl